src/llm_vm/completion/test_optimize.py

```diff
@@ -99,23 +99,27 @@ def create_jsonl_file(data_list: list, file_name: str, compress: bool = True) ->
     return file_name, open(file_name, "rb")
 '''
 
-if __name__ == "__main__":
+def run_test_stub():
     try:
         load_dotenv()
     except:
         pass
-    openai.api_key = os.getenv('OPENAI_KEY')
-    anarchy_key = os.getenv('ANARCHY_KEY')
+    openai_api_key = os.getenv('OPENAI_API_KEY')
```

Comment:
i dont know why it wasnt working but these changes fixed it up... and also use the env key that openai expects

Reply:
this might break your setup @abhigya-sodani .. :)